src/main/kotlin/no/nav/tiltakspenger/saksbehandling/behandling/domene/Behandling.kt

```diff
@@ -42,7 +42,7 @@ sealed interface Behandling {
     val iverksattTidspunkt: LocalDateTime?
     val sendtTilDatadeling: LocalDateTime?
     val sakId: SakId
-    val oppgaveId: OppgaveId?
+    val oppgaveId: OppgaveId? // søknadOppgaveId?
```

Comment:
Tenker den er grei som den er. Den kan jo basere seg både på oppgaven som ble laget for søknaden, men også for oppgaven som ble laget av søknadsbehandling nr. 2.

Sannsynligvis fjerner vi den på sikt, siden de ikke vil ha den funksjonaliteten.